README.md

```diff
@@ -190,8 +190,9 @@ delimited binary protocol buffers or in the simple flat text format
 (both in version 0.0.4, see the
 [data exposition format specification](https://docs.google.com/document/d/1ZjyKiKxZV83VI9ZKAXRGKaUKK2BIWCT7oiGBKDBpjEY/edit?usp=sharing)).
 Discrimination between the two variants is done via the `Content-Type`
-header. (In case of an unknown value for `Content-Type`, the text
-format is tried as a fall-back.)
+header. (Use the value `application/vnd.google.protobuf; proto=io.prometheus.client.MetricFamily; encoding=delimited`
+if the messages are protobuf encoded, otherwise the text format is tried
+as a fall-back.)
```

Comment:
For consistency, I'd suggest: "...if the messages are using the protobuf format, ..."

Or' if you want to be super precise: "... if the messages are using the protobuf format with _delimited_ encoding, ..."